src/dcp/dcp.c

```diff
@@ -231,6 +234,23 @@ int main(int argc, char** argv)
                     }
                 }
                 break;
+            case 'e':
+                copy_xattrs_used = 1;
+                if (!strcmp(optarg,"none")) {
+                    mfu_copy_opts->copy_xattrs = XATTR_COPY_NONE;
+                } else if (!strcmp(optarg,"non-lustre")) {
+                    mfu_copy_opts->copy_xattrs = XATTR_SKIP_LUSTRE;
+                } else if (!strcmp(optarg,"libattr")) {
+                    mfu_copy_opts->copy_xattrs = XATTR_USE_LIBATTR;
+                } else if (!strcmp(optarg,"all")) {
+                    mfu_copy_opts->copy_xattrs = XATTR_COPY_ALL;
+                } else {
+                    if (rank == 0) {
+                        MFU_LOG(MFU_LOG_ERR, "Unrecognized option for --copy-xattrs");
```

Comment:
It error message would be more useful if it printed the actual bad option inside single quotes, in case it is accidentally a filename or similar.